core/translate/mod.rs

```diff
@@ -600,6 +600,10 @@ fn update_pragma(
             query_pragma("wal_checkpoint", header, program)?;
             Ok(())
         }
+        PragmaName::PageCount => {
+            query_pragma("page_count", header, program)?;
+            Ok(())
```

Comment:
Why do we have a transform from `PragmaName` to `str` to then `PragmaName` again. Quite unnecessary. (no need to solve it here)

Reply:
https://github.com/tursodatabase/limbo/issues/823 created an issue